### PR TITLE
Do not use else after a return in an if clause

### PR DESCRIPTION
**Pull request by blink1073**
Also use string literals.

### CODE REVIEW

**Comment by sccolbert** [comment]
don't merge this until we know why tsc is failing.

File: src/output-area/model.ts [hunk 1]
```diff
@@ -72,19 +62,19 @@ class OutputBaseModel {
...
+   * Metadata about the output.
+   */
   metadata: any;
 
   /**
-   * Output type
+   * The output type.
    */
-  outputType: OutputType = OutputType.DisplayData;
+  outputType: "display_data";
```
Comment (sccolbert) [inline src/output-area/model.ts]: `outputType: OutputType = "display_data"`

File: src/output-area/model.ts [hunk 2]
```diff
@@ -92,33 +82,30 @@ class DisplayDataModel extends OutputBaseModel {
...
    */
-  StdOut,
+  executionCount: number; // this is also a property on the cell?
 
   /**
-   * The "stderr" stream name from the message spec.
+   * The output type.
    */
-  StdErr
+  outputType: "execute_result";
```
Comment (sccolbert) [inline src/output-area/model.ts]: ditto

File: src/output-area/model.ts [hunk 3]
```diff
@@ -135,14 +122,14 @@ class StreamModel extends OutputBaseModel {
   text: string;
 
   /**
-   * Output type
+   * The output type.
    */
-  outputType: OutputType = OutputType.Stream;
+  outputType: "stream";
```
Comment (sccolbert) [inline src/output-area/model.ts]: ditto

File: src/output-area/model.ts (hunk 1 above)
Comment (blink1073) [inline src/output-area/model.ts]: Even though it is defined as such in the base model?

Comment (sccolbert) [inline src/output-area/model.ts]: are you declaring, or initializing here?

Comment (blink1073) [inline src/output-area/model.ts]: Both, overriding the base string literal, which becomes the value.

Comment (sccolbert) [inline src/output-area/model.ts]: This is only a declaration. It's not an initialization. You have to use `=` and actually assign it something or the actual value will be `undefined`.

**Comment by sccolbert** [comment]
:shipit:

**Pull request merged**